cohortextractor/query_engines/base_sql.py

```diff
@@ -175,7 +175,7 @@ def get_output_column_name(node):
         elif isinstance(node, (ValueFromRow, Column)):
             return node.column
         else:
-            raise TypeError(f"Unhandled type: {node}")
+            assert False, f"Unhandled type: {node}"
```

Comment:
@inglesp And you claim that the codebase is well typed? :rofl:

Reply:
I stand by that!  There's just an undefined interface that `ValueFromAggregate` and `ValueFromRow` implement.

What I meant is that this kind of instance checking seems very self-contained and doesn't bleed through the codebase.